src/Service/DigitalPostHelper.php

```diff
@@ -196,13 +176,7 @@ private function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setRequired([
             'digital_post_options',
+            'restriction_options',
```

Comment:
You could validate sub-values as well (see https://github.com/itk-dev/naevnssekretariatet/blob/develop/src/Service/LogEntryHelper.php#L27-L36).